templates/components/common/rating.html

```diff
@@ -0,0 +1,7 @@
+<div class="c-rating">
+    {{> components/common/star index='1'}}
+    {{> components/common/star index='2'}}
+    {{> components/common/star index='3'}}
+    {{> components/common/star index='4'}}
+    {{> components/common/star index='5'}}
```

Comment:
Also, add information about the rating for screen readers. It could be a span visually hidden with the text `x.x out of 5 stars`